diff --git a/build/src/start/server/metro/TerminalReporter.js b/build/src/start/server/metro/TerminalReporter.js
index abc1234..def5678 100644
--- a/build/src/start/server/metro/TerminalReporter.js
+++ b/build/src/start/server/metro/TerminalReporter.js
@@ -1,5 +1,5 @@
 function _terminalReporter() {
-    const data = /*#__PURE__*/ _interopRequireDefault(require("metro/src/lib/TerminalReporter"));
+    const data = /*#__PURE__*/ _interopRequireDefault(require("metro/private/lib/TerminalReporter"));
     _terminalReporter = function() {
         return data;
     };
